addons/mail/static/src/core/common/message_model.js

```diff
@@ -175,6 +174,7 @@ export class Message extends Record {
     create_date = Record.attr(undefined, { type: "datetime" });
     /** @type {luxon.DateTime} */
     write_date = Record.attr(undefined, { type: "datetime" });
+    needaction = false;
```

Comment:
Default should probably be undefined, we can't tell what is the value until the server sends it.